docs/cmake-project-view.md

```diff
@@ -0,0 +1,116 @@
+# CMake Project Outline View
```

Comment:
This shouldn't be "CMake Project Status View". But since this is also status bar settings, should this be titled something else, and the file named something else? Like "CMake Options Visibility Configuration" maybe and the file be `cmake-options-config.md`? idk just a suggestion